x-pack/solutions/security/plugins/security_solution/public/data_view_picker/redux/types.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { DataViewSpec } from '@kbn/data-views-plugin/common';
+
+export interface ScopedDataViewSelectionState {
+  dataView: DataViewSpec | null;
+  /**
+   * There are several states the picker can be in internally:
+   * - pristine - not initialized yet
+   * - loading
```

Comment:
`loading - the dataView instance is loading with all associated fields, runtime fields, etc...`